database/testbed/docker-compose.yml

```diff
@@ -1,7 +1,10 @@
-database:
-  build: ./database
-  ports:
-    - "15432:5432"
-  environment:
-    POSTGRES_PASSWORD: postgres
+version: '3.2'
```

Comment:
The `version` property is obsolete. To avoid the warning

```
WARN[0000] /path/to/postnas-inspire-alignments/database/testbed/docker-compose.yml: `version` is obsolete
```

when using a recent Docker version, it should be removed.

Reply:
Thanks for the hint. I changed this.